README.md

```diff
@@ -130,4 +138,4 @@ This pipeline uses code and infrastructure developed and maintained by the [nf-c
 >
 > Philip Ewels, Alexander Peltzer, Sven Fillinger, Harshil Patel, Johannes Alneberg, Andreas Wilm, Maxime Ulysse Garcia, Paolo Di Tommaso & Sven Nahnsen.
 >
-> _Nat Biotechnol._ 2020 Feb 13. doi: [10.1038/s41587-020-0439-x](https://dx.doi.org/10.1038/s41587-020-0439-x).
\ No newline at end of file
+> _Nat Biotechnol._ 2020 Feb 13. doi: [10.1038/s41587-020-0439-x](https://dx.doi.org/10.1038/s41587-020-0439-x).
```

Comment:
I of course still need to go through and eventually do an additional round of README overhaul, describing all the outputs, and citing all software used.

Reply:
would be good to post an issue for this if there isn't one already

Reply:
There is #18. Just increased its scope a little bit, it was focused on the output directory structure